front/src/applications/stdcm/components/StdcmResults/SendToRailwayManagerModal.tsx

```diff
@@ -0,0 +1,416 @@
+  type StdcmResultsOperationalPoint,
+} from 'applications/stdcm/types';
+import {
+  osrdRailwayManagerApi,
+  type BodyLastMinuteRequest,
+} from 'common/api/osrdRailwayManagerApi';
+import MAIN_API from 'config/config';
+import {
+  createLinkedTrainPayload,
+  createSimilarTrainPayload,
+  getStopDurationTime,
+  transformStepsToApiFormat,
+  validateTolerance,
+} from 'modules/SimulationReportSheet/utils/formatSimulationReportSheet';
+import type { StdcmPathStep } from 'reducers/osrdconf/types';
+import type { DeploymentSettings } from 'utils/hooks/useDeploymentSettings';
+import { kmhToMs, tToKg } from 'utils/physics';
+
+import StdcmSimulationReportSheet from './StdcmSimulationReportSheet';
+
+dayjs.extend(utc);
+
+type SendToRailwayManagerModalProps = {
+  onClose: () => void;
+  consist: StdcmSimulationInputs['consist'];
+  stdcmData: StdcmSuccessResponse;
+  linkedTrains: StdcmSimulationInputs['linkedTrains'];
+  simulationReportSheetNumber: string;
+  operationalPointsList: StdcmResultsOperationalPoint[];
+  simulationSheetLogo?: string;
+  similarTrains: SimilarTrainWithSecondaryCode[];
+  deploymentSettings?: DeploymentSettings;
+};
+
+type Option = { value: string; label: string };
+
+const SendToRailwayManagerModal = ({
+  consist,
+  onClose,
+  stdcmData,
+  linkedTrains,
+  simulationReportSheetNumber,
+  operationalPointsList,
+  simulationSheetLogo,
+  similarTrains,
+  deploymentSettings,
+}: SendToRailwayManagerModalProps) => {
+  const { t } = useTranslation('stdcm', { keyPrefix: 'simulation.results' });
+
+  const steps = stdcmData?.simulationPathSteps ?? [];
+
+  const firstStep = steps[0] as Extract<StdcmPathStep, { isVia: false }>;
+  const lastStep = steps[steps.length - 1] as Extract<StdcmPathStep, { isVia: false }>;
+
+  const scheduledDepartureTime = dayjs(stdcmData.departure_time);
+  const departureTime = dayjs(firstStep.arrival);
+
+  const intermediatePoints = useMemo(() => steps.slice(1, -1), [steps]);
+
+  const [pathType, setPathType] = useState<Option | undefined>(undefined);
+  const [comment, setComment] = useState('');
+  const [csCode, setCsCode] = useState('');
+  const diffMinutes = scheduledDepartureTime.diff(departureTime, 'minute');
+
+  const baseToleranceBefore = firstStep.tolerances.before.total('minute');
+  const baseToleranceAfter = firstStep.tolerances.after.total('minute');
+
+  const [beforeTolerance, setBeforeTolerance] = useState(
+    (baseToleranceBefore + diffMinutes).toString()
+  );
+  const [afterTolerance, setAfterTolerance] = useState(
+    (baseToleranceAfter - diffMinutes).toString()
+  );
+  const [pathTypeError, setPathTypeError] = useState(false);
+  const pdfBlobRef = useRef<Blob | null>(null);
+
+  const [sendLastMinuteRequest, { isLoading, isSuccess }] =
+    osrdRailwayManagerApi.useSendLastMinuteRequestSendLastMinuteRequestPostMutation();
+
+  const convoyDetails = useMemo(
+    () => [
+      { label: t('modal.compositionCode'), value: consist?.speedLimitByTag },
```

Comment:
I think it would be nice to handle all the fields the same way
```suggestion
      { label: t('modal.compositionCode'), value: consist?.speedLimitByTag ?? '-' },
```

Reply:
Done